tests/unit/s2n_security_policies_test.c

```diff
@@ -174,6 +174,8 @@ int main(int argc, char **argv)
             "default_tls13",
             "test_all",
             "test_all_tls13",
+            "20190801",
+            "20190802"
```

Comment:
Do we expect this list to always contain all the security policy strings? If so, is that enforced anywhere?

Reply:
This list only contains `tls13` policies for testing purposes.

Reply:
But do you expect people to add new tls13 policies to this list if they create them, and is that enforced?

Reply:
This list is used to test `s2n_security_policy_supports_tls13` function, it is not an absolutely necessary to add all the new tls13 policies to it. It is okay for a subset of the policies to be present in the list and does not require for it to be enforced.